lib/msgraph/paginated.go

```diff
@@ -54,7 +54,14 @@ func iterateSimple[T any](c *Client, ctx context.Context, endpoint string, f fun
 func (c *Client) iterate(ctx context.Context, endpoint string, f func(json.RawMessage) bool) error {
 	uri := *c.baseURL
 	uri.Path = path.Join(uri.Path, endpoint)
-	uri.RawQuery = url.Values{"$top": {strconv.Itoa(c.pageSize)}}.Encode()
+	uri.RawQuery = url.Values{
+		"$top": {
+			strconv.Itoa(c.pageSize),
+		},
+		"$expand": {
+			"memberOf",
+		},
```

Comment:
I think the impact will be minor, if noticeable at all, since the membership is just `id` and `type`. And the number of items from an `$expand` parameter is limited to 20 (which I've created a separate issue for https://github.com/gravitational/access-graph/issues/1400). But I'll add an optional `query` parameter just in case I'm wrong.